components/collective-page/index.js

```diff
@@ -273,10 +282,30 @@ class CollectivePage extends Component {
     }
   }
 
+  createNotification = (type, notificationCollectiveName) => {
+    this.setState({ applicationNotification: true, notificationType: type, notificationCollectiveName });
```

Comment:
What about making this function agnostic about the notification content? Pros:
- This component should ideally be only about what's generic between collectives. Having some special logic and error messages for a very specific host action in it seems to go out of it's base responsibility.
- We'll be able to reuse the notification easily in other sections easily
- Simpler state

```suggestion
  createNotification = (type, message) => {
    this.setState({notification: { type, message } });
```

Reply:
That's a good idea!